packages/@sanity/default-login/README.md

```diff
@@ -2,14 +2,49 @@
 
 Let a user log into Sanity, and get access to the child content.
 
+
+## Configuration
+
+By default this component will ask the Sanity API and get a list of providers, which the user can login with.
+
+However it is possible for enterprise customers to supply their own third party authentication server as a provider as well.
+
+This is done through configuring the studio with the file ``./config/@sanity/default-login.json``
+
+Example:
+
+```
+{
+  // Your custom provider implementations
+  "customProviders": [
```

Comment:
For the sake of brevity, could this key be just `providers`?

Reply:
That was my first name, however it was pretty confusing, because they an be appended to or replace the existing providers. So what is what.